client/starwhale/core/model/model.py

```diff
@@ -230,20 +241,23 @@ def eval_user_handler(
         if job_name not in _jobs:
             raise RuntimeError(f"job:{job_name} not found")
 
+        project = os.getenv(SWEnv.project)
+        if project is None:
+            project = SWCliConfigMixed().current_project
```

Comment:
in the cloud task container, what is the current_project?

Reply:
First, use Project URI to parse, so, if a user sends or sets env for the project, use it. if exec without --project, use current project(get from starwhale config)